packages/select/src/FilterMultiSelect/components/Trigger/FilterTriggerButton/FilterTriggerButton.tsx

```diff
@@ -7,12 +7,14 @@ export type FilterTriggerButtonProps = {
   label: string
   selectedOptionLabels: string[]
   classNameOverride?: string // TODO: migrate it to use OverrideClassName<T> and omit the props controlled by React-Aria
+  characterLimit?: number
```

Comment:
Hmm...given this prop name, I would expect it to cap the label length, however the functionality would actually still display a full length long string if it were the first option to display.

Perhaps clarify with @jmitsios if we want to actually truncate the first label if it exceeds the character limit? If not, we should try and think of a better name. 

It could benefit from a description regardless too :)

Reply:
We discussed this in the kickoff and the first label regardless of its char length should always display. We also discussed there isn't an obvious name to convey that obscurity. I will add a description which will defs help.

Reply:
Hmm yeah that's hard 🤔 I can't think of anything better currently either. A description will do for now :)

Reply:
> The first label regardless of its char length should always display

Definitely agree but now I feel like we should probably account for how long the label is in this calculation (if that's not happening already?) 

Is this too wordy: `labelCharacterLimitBeforeTruncate`?

Reply:
If it adds clarity, there's nothing wrong with a long variable imo. Shippit